### PR TITLE
Xiyue enddate create form

### LINKED ISSUE #66: Add Enddate Field to Create Form
Allow admin to create a commons with end date.

### PR DESCRIPTION
**Pull request by XYWayne**
# Overview

Allow admins to set end date when creating a commons. Then it will be displayed on play page.

Closes #66 

# Details

https://user-images.githubusercontent.com/59850013/157999365-6acefe0c-5200-477d-8ca4-c3cf94409414.mov

### CODE REVIEW

**Review by vChase42 — DISMISSED** [review]
Shows up for me on Heroku and localhost, LGTM

**Review by vChase42 — DISMISSED** [review]
LGTM

**Review by vChase42 — DISMISSED** [review]
LGTM

**Review by isaacchin — DISMISSED** [review]
LGTM

File: frontend/src/fixtures/commonsFixtures.js [hunk 1]
```diff
@@ -4,37 +4,34 @@ const commonsFixtures = {
             "id": 5,
             "name": "Seths Common",
             "day": 5,
-            "startDate": "2022-03-05T15:50:10",
-            "endDate": "2023-03-05T15:50:10",
+            "startDate": "6/10/2021",
```
Comment (pconrad) [inline frontend/src/fixtures/commonsFixtures.js:7]: Per Xiyue, these should be ISO dates; that's what's returned by the backend.

So let's make the fixtures match

**Review by pconrad — CHANGES_REQUESTED** [review]
LGTM except for restoring the iso-dates

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**